Automattic/newspack-theme
# Remove styles associated with the already-removed featured image header

### All Submissions:


### Changes proposed in this Pull Request:

This PR just cleans up some Sass/styles related to the featured image header that was removed in https://github.com/Automattic/newspack-theme/commit/5138fae3c56e1113d9124300c90260d1618e7732. 

### How to test the changes in this Pull Request:

Probably the best two things to check to make sure this doesn't introduce issues are:

1. Double-check that all of the removed styles start with the class `.featured-image` -- this would help make sure I haven't removed anything I shouldn't have. 
2. Double-check that the Sass files build without errors.

### Other information:
